build/CoreGraphics/lib/CoreGraphicsLib.vcxproj

```diff
@@ -55,6 +55,10 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFStream.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGPDFString.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGShading.mm" />
+    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\D2DWrapper.mm" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\include\CoreGraphics\D2DWrapper.h" />
```

Comment:
+1, this should be a private header.  Include\CoreGraphics should only have headers that are public (and in reference platform).  

---
In reply to: [82067697](https://github.com/Microsoft/WinObjC/pull/1085#discussion_r82067697) [](ancestors = 82067697,82040170,82025651)